mixins/inputMessage.js

```diff
@@ -14,15 +15,17 @@ export default {
   },
 
   shouldShowError () {
-    let errors = this.props.errors || []
+    let propErrors = this.props.errors || []
+    const stateErrors = get(this, 'state.errors') || []
```

Comment:
Why? 
Is this working as a try on `state`, as the component may not be stateful?